testrunner-running/src/main/scala/edu/illinois/cs/testrunner/execution/TestListener.java

```diff
@@ -35,7 +37,16 @@ public void testIgnored(Description description) throws Exception {
 
     @Override
     public void testStarted(Description description) throws Exception {
-        times.put(JUnitTestRunner.fullName(description), System.nanoTime());
+        String fullTestName = JUnitTestRunner.fullName(description);
+        times.put(fullTestName, System.nanoTime());
+
+        String phase = Configuration.config().getProperty("statecapture.phase", "");
+        if (Configuration.config().getProperty("replay.dtname").equals(fullTestName)) {
```

Comment:
Can we rename this property to be something like "statecapture.testname"? I would like this state-capturing logic to be relatively self-contained and not tied so closely with the eventual users of testrunner to capture state and such.